transport/src/main/java/io/netty/channel/nio/NioEventLoopGroup.java

```diff
@@ -84,6 +85,16 @@ public void rebuildSelectors() {
     @Override
     protected EventExecutor newChild(
             ThreadFactory threadFactory, Object... args) throws Exception {
-        return new NioEventLoop(this, threadFactory, (SelectorProvider) args[0]);
+        return new NioEventLoop(this, threadFactory, (SelectorProvider) args[0], backoffStrategy());
+    }
+
+    /**
+     * Backoff Strategy which is called every time a {@link #newChild(ThreadFactory, Object...)} is
+     * created. Can be overridden if a custom backoff strategy should be used.
+     *
+     * @return {@link NoopBackoffStrategy} as the default.
+     */
+    public BackoffStrategy backoffStrategy() {
+        return new NoopBackoffStrategy();
```

Comment:
> we'd need some kind of factory 

Yip makes sense if the `BackoffStrategy` has state. However this approach seems more portable if we eventually want to consider this patch for EpollEventLoop too.